plugins/CoreAdminHome/templates/trackingCodeGenerator.twig

```diff
@@ -62,6 +63,10 @@
      <p>{{ 'CoreAdminHome_SinglePageApplicationDescription'|translate('<a href="https://developer.matomo.org/guides/spa-tracking" rel="noreferrer noopener" target="_blank">','</a>')|raw }}</p>
 </div>
 
+<div piwik-content-block vue-entry="CoreHome.ContentBlock" content-title="{{ 'SitesManager_SiteWithoutDataCloudflare'|translate }}" anchor="cloudflare">
```

Comment:
I don't think that you want to use `piwik-content-block` in conjunction with `vue-entry="CoreHome.ContentBlock"`. I believe that you want to use one or the other, not both.

Reply:
@snake14 I added this for portability when we merge again with 4.x-dev, but will make a note to do that there

Reply:
Removed